File: translation_output/misleading_test/gpt-4/avatar/python/java/atcoder_ABC125_A.java
Here's the equivalent Java code for the provided Python code:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int A = sc.nextInt();
        int B = sc.nextInt();
        int T = sc.nextInt();
        int ans = (T / A) * B;
        System.out.println(ans);
        sc.close();
    }
}
```

This Java program replicates the behavior of the Python code, where it takes three integers as input and calculates the result of integer division of `T` by `A`, multiplied by `B`, and then prints the outcome. The `Scanner` class is used for input operations in Java which is similar to `input().split()` in Python.

File: Results/DER/Translation/translation_output/misleading_test/gpt-3.5/avatar/java/python-sanitized/atcoder_ABC125_A.py
def atcoder_ABC125_A():
    n, k, m, l, a, b, c = map(int, input().split())
    k = 0
    for i in range(1, n//a + 1):
        k += b
    return k

result = atcoder_ABC125_A()
print(result)